iron-menu-behavior.html

```diff
@@ -133,6 +133,25 @@
      * @param {KeyboardEvent} event A KeyboardEvent.
      */
     _focusWithKeyboardEvent: function(event) {
+      // Make sure that the key pressed is not a modifier key. The list of keys
+      // has been taken from https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState
```

Comment:
(For reference, [this is where `getModifierState` was added in WebKit](https://trac.webkit.org/changeset/206725/webkit) and that revision was included in release [602.2.14.0.5](https://trac.webkit.org/changeset/206726/webkit), which is at least in [Safari 10.0.2](https://en.wikipedia.org/wiki/Safari_version_history).)